database/migrations/2020_03_23_000000_add_referrer_user_id_to_signups_and_posts_tables.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddReferrerUserIdToSignupsAndPostsTables extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('signups', function (Blueprint $table) {
+            $table->string('referrer_user_id')->nullable()->index()->after('source_details');
+        });
+        Schema::table('posts', function (Blueprint $table) {
+            $table->string('referrer_user_id')->nullable()->index()->after('details');
+            $table->index(['referrer_user_id', 'type']);
```

Comment:
What does this do again?

Reply:
Adds a [compound index](https://laravel.com/docs/5.8/migrations#creating-indexes) so we can speed up queries to find all of the posts with `type=voter-reg` and `referrer_user_id` equal to our authenticated user, when displaying the total # of beta's they got registered to vote via their OVRD. Refs [Slack comment](https://dosomething.slack.com/archives/CP2D7UGAU/p1584987935004300?thread_ts=1584987535.004000&cid=CP2D7UGAU)

Reply:
Ah noice! 👏